packages/rum/README.md

```diff
@@ -73,9 +73,9 @@ datadogRum.init({
   //  service: 'my-web-application',
   //  env: 'production',
   //  version: '1.0.0',
-  sampleRate: 100,
+  sessionSampleRate: 100,
   premiumSampleRate: 100, // if not included, the default is 100
-  trackInteractions: true,
+  trackUserInteractions: true,
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏You changed the doc corresponding to code snippet to be used before v4.20.0 and v4.10.2. It should stay the same here.

<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏maybe add a similar section "before v4.30.0"